### PR TITLE
Feature/update tutorials

### PR DESCRIPTION
**Pull request by DanielReid**

### CODE REVIEW

File: tutorials/01_manual_input.tex [hunk 1]
```diff
@@ -38,108 +38,94 @@ \subsection*{Create a new workspace}
...
 
 \subsection*{Criteria and favorability}
-\noindent \leftpointright \, The effects table for lixisenatide divides the criteria into favourable and unfavourable effects. To enable this classification, check the ‘Use favorability’ box below the ‘Criteria’ heading.
-\newline
 
 \noindent \faGraduationCap \, Criteria are the clinical efficacy and safety outcomes in terms of which the treatments under consideration are evaluated. A criterion has a name (a short identifier, such as  “HbA1c” or “Nausea”), and a description. The description specifies how the effect of a treatment on the selected outcome variable is measured statistically. For example, for “HbA1c” this could be “mean change of HbA1c from baseline” or “proportion of patients with a normal HbA1c value at week 24”.
 \newline
 
 \noindent \faGraduationCap \, For regulatory purposes, criteria can additionally be classified as favourable or unfavourable, with favourable criteria being ones where the new treatment under consideration outperforms the status quo.
```
Comment (kzalite) [inline tutorials/01_manual_input.tex:54]: missing \newline at the of this paragraph

File: tutorials/01_manual_input.tex [hunk 2]
```diff
@@ -38,108 +38,94 @@ \subsection*{Create a new workspace}
...
 
-\subsection*{Add the HbA1c criterion}
-\noindent \leftpointright \, Click the ‘Add Criterion’ button and fill in the dialog as specified in Figure \ref{fig:crit1}.
+\noindent \leftpointright \, The effects table for lixisenatide divides the criteria into favourable and unfavourable effects. To enable this classification, ensure that the ‘Use favorability’ box below the ‘Criteria’ heading is checked.
```
Comment (chinook25) [inline tutorials/01_manual_input.tex:56]: 'Use favourability'

File: tutorials/01_manual_input.tex [hunk 3]
```diff
@@ -38,108 +38,94 @@ \subsection*{Create a new workspace}
...
-    \caption{Add criterion dialog for Nausea.}
-	\label{fig:crit2}
-\end{figure}
-
-\noindent \leftpointright \, Click the ‘Add data source button and enter “Pooled data from all phase 2/3 controlled studies” as the the reference for this data source.  There are no further data sources for this criterion.
+\noindent \leftpointright \, Change the reference to “Pooled data from all phase 2/3 controlled studies”.  There are no further references for this criterion.
 
 \subsection*{Add lixisenatide and placebo as alternatives}
-\noindent \leftpointright \, Under the ‘Alternatives’ header, click the ‘Add Alternative’ button and fill in “lixisenatide” in the ‘Title’ text box. Click the ‘Add’ button. Repeat the process and add a “placebo” alternative.
+\noindent \leftpointright \, Rename 'alternative 1' to by clicking on it and entering 'lixisenatide' in the 'Title' text box. Rename 'alternative 1' to 'placebo'.
```
Comment (kzalite) [inline tutorials/01_manual_input.tex:77]: Delete 'to' before by, no more, no less. 'To' shall be the word thy shalt delete.

File: tutorials/01_manual_input.tex [hunk 4]
```diff
@@ -38,108 +38,94 @@ \subsection*{Create a new workspace}
...
 
 \subsection*{Add Strength of evidence/Uncertainties}
 \noindent \faGraduationCap \, The ‘Uncertainties’ and ‘Strength of evidence’ fields describe any strengths and weaknesses of the study design, quality of the data collection and analysis, generalizability of the study findings, etc.
 \newline
 
-\noindent \leftpointright \, We will add the following Uncertainty to the ‘HbA1c/Pooled data’ data source: “The effect of lixisenatide was more pronounced in Asian patients compared to Caucasian patients. The lower effect in Caucasians is partly explained by a large placebo effect especially in some geographical regions.” To do so, click the edit button in the ‘Strength of evidence/Uncertainties’ cell and fill the Uncertainties text box.
+\noindent \leftpointright \, We will add the following Uncertainty to the ‘HbA1c/Pooled data’ data source: “The effect of lixisenatide was more pronounced in Asian patients compared to Caucasian patients. The lower effect in Caucasians is partly explained by a large placebo effect especially in some geographical regions.” To do so, click the 'Unc' text in and fill the text box.
```
Comment (kzalite) [inline tutorials/01_manual_input.tex:101]: Delete 'in' at the end of the paragraph after 'Unc' text

**Review by kzalite — CHANGES_REQUESTED** [review]

**Pull request merged**